client/src/components/questions/Question.jsx

```diff
@@ -80,6 +82,7 @@ const Question = ({ questionData, productName }) => {
 
   var submitAnswer = (e, photoArray) => {
     e.preventDefault();
+    console.log(photoArray);
```

Comment:
Did this work? Was photoArray filled with anything?

Reply:
oh, and console.log

Reply:
yes it was filled with object URLs when an image was uploaded. Found out that the only solution to this is to store the object URLs on an image hosting site.